### ISSUE TITLE
Accessibility Bug: No accessible name for the link on the Topcoder logo

### ISSUE THREAD

**Issue by CorgiDev** [topcoder-platform/Training-Challenges#265]
Steps to Reproduce:

1. Open Dev Tools for browser.
2. Inspect the Topcoder logo in the top, left-hand corner of the page.
3. Neither the url and the svg have a proper labeling or alt text to help identify it's purpose to users of assistive technology.

Actual Result: 
Inspecting the logo reveals no alt text or proper aria that would make a label for it accessible to assistive technologies. When a screen reader hits the logo link, it just reads off the url, which not considered best practice or considered an accessible label.

Expected Result: 
Using assistive technology should cause the logo to be identified with an simple, descriptive name, like just Topcoder.

Screenshots or screencast URL: <URL of screenshots/screencast of the issue>
Device/OS/Browser Information: Desktop / Windows 11 (Build 22454) / Edge (Version 94.0.992.31)

A simple solution to this would be to add the following attributes to the SVG image that makes up the logo:

- `aria-hidden="true"
- `aria-role="presentation"`
- `alt=" "`

Then add the following attribute to the link itself: `aria-label="Topcoder"`

**Comment by nithyaasworld** [comment]
Hi @CorgiDev The reviewer should be able to reproduce the issue just by following the steps given in the 'Steps to reproduce' section. Can you please update the steps with more details on what URL to go first before inspecting the Topcoder logo?

**Comment by CorgiDev** [comment]
@nithyaasworld Updated steps with URL and added screenshots. I think this one is similar to #266 , in that it is a component shared multiple, if not all, pages.
